src/main/java/seedu/mtracker/filemanager/ForexDecoder.java

```diff
@@ -12,16 +15,57 @@
     public static final int EXIT_PRICE_INDEX = 6;
     public static final int FOREX_EXPIRY_INDEX = 7;
     public static final int FOREX_REMARKS_INDEX = 8;
+    protected static double decodedEntryPrice;
+    protected static double decodedExitPrice;
+    protected static LocalDate decodedExpiry;
+    protected static String decodedRemarks;
+
+    public static boolean isValidEntryAndExitPrice(String[] textSegment) {
+        return Validate.isValidPrice(textSegment[ENTRY_PRICE_INDEX])
+                && Validate.isValidPrice(textSegment[EXIT_PRICE_INDEX]);
+    }
```

Comment:
I think you shld split this up. To make specific errors we have to create individual methods for each attribute